abakus-statutter/08-generalforsamling.tex

```diff
@@ -94,12 +94,11 @@ \subsubsection{}
 \subsection{Valg}
 \subsubsection{}
 Samtlige stillinger i Hovedstyret velges av Generalforsamling ved
-alminnelig flertall. Så lenge ingen kandidater oppnår alminnelig flertall
-fjernes den kandidaten med lavest antall stemmer og det stemmes på nytt. Blanke
+alminnelig flertall. Valget avholdes med Preferansevalg (jfr. §0.6). Blanke
```

Comment:
§0.6 has not been added in this PR. Is this intentional?

Reply:
Good point! Where's my statutt ⭕

Reply:
Fixed!